exec/java-exec/src/main/java/org/apache/drill/exec/store/sys/zk/ZkEStore.java

```diff
@@ -17,40 +17,25 @@
  */
 package org.apache.drill.exec.store.sys.zk;
 
+import java.io.IOException;
+
 import org.apache.curator.framework.CuratorFramework;
 import org.apache.drill.exec.store.sys.EStore;
 import org.apache.drill.exec.store.sys.PStoreConfig;
 import org.apache.zookeeper.CreateMode;
 
-import java.io.IOException;
-
 /**
  * Implementation of EStore using Zookeeper's EPHEMERAL node.
  * @param <V>
  */
-public class ZkEStore<V> extends ZkAbstractStore<V> implements EStore<V>{
+public class ZkEStore<V> extends ZkAbstractStore<V> implements EStore<V> {
 
-  public ZkEStore(CuratorFramework framework, PStoreConfig<V> config) throws IOException{
+  public ZkEStore(CuratorFramework framework, PStoreConfig<V> config) throws IOException {
     super(framework,config);
   }
 
   @Override
-  public void delete(String key) {
-    try {
-      if (framework.checkExists().forPath(p(key)) != null) {
-        framework.delete().forPath(p(key));
```

Comment:
Previously the node was not rebuilt for ZkEStore (see `childrenCache.rebuildNode(withPrefix(key))` in ZkAbstractStore). Is this change necessary?